packages/test-data/tests-unit/variables/legacy/variable-advanced.css

```
@@ -0,0 +1,56 @@
+
+.units-extended {
+  width: 1px;
+  same-unit-as-previously: 1px;
+  square-pixel-divided: 1px;
+  odd-unit: 2;
+  percentage: 500%;
+  pixels: 500px;
+  conversion-metric-a: 30mm;
+  conversion-metric-b: 3cm;
+  conversion-imperial: 3in;
+  custom-unit: 420octocats;
```

Stylelint (17.14.0)

[error] Unknown unit "octocats" (unit-no-unknown)

(unit-no-unknown)